dependencies/socketw/sw_base.h

```diff
@@ -18,7 +18,9 @@
 
 #include "sw_internal.h"
 
-#include <unistd.h>
+#if !(_MSC_VER == 1900) // Visual Studio 2015 has no <unistd.h>
+#   include <unistd.h>
```

Comment:
Right. I wasn't sure whether MS doesn't have any supplement in other versions. I'll update the condition.